kratos/sources/geometrical_object.cpp

```diff
@@ -0,0 +1,28 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher (https://github.com/philbucher)
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/geometrical_object.h"
+#include "includes/kratos_flags.h"
+
+namespace Kratos {
+
+bool GeometricalObject::IsActive() const 
+{
+    return IsDefined(ACTIVE) ? Is(ACTIVE) : true;

```

Comment:
In checking for true, the `IsDefined` call is redundant. The `IsDefined` shall be use if we want to check for false in which we don't know if the flag is unset or is really set to false.

Reply:
I am sorry I am a bit lost :sweat_smile: 
Can you please post your suggestion?

Reply:
It is correct! Sorry for the noise...